docusaurus/docs/getting-started.mdx

```diff
@@ -22,44 +24,54 @@ Plugin tools consists of two packages:
   queryString="current-package-manager"
 />
 
-Follow the prompts to scaffold a Grafana plugin. [See here](./creating-a-plugin.mdx) for detailed information about creating a plugin.
+For next steps, see [Creating a plugin](./creating-a-plugin.mdx) 
 
 :::note
 
-Plugins previously scaffolded with `@grafana/toolkit` can be migrated. Please see [this migration guide](./migrating-from-toolkit).
+If you have previously built a plugin with `@grafana/toolkit`, you can use our plugin tools to simplify migration. For more information, refer to the [Migration guide](./migrating-from-toolkit).
 
 :::
 
-### What you'll need
+### Before you begin 
 
-- [Go](https://go.dev/doc/install) version 1.18 or above
-- [Mage](https://magefile.org/)
-- [Node.js](https://nodejs.org/en/download/) version 16 or above:
-  - When installing Node.js, you are recommended to check all checkboxes related to dependencies.
-- Optionally [Yarn 1](https://classic.yarnpkg.com/lang/en/docs/install) or [PNPM](https://pnpm.io/installation)
-- [Docker](https://docs.docker.com/get-docker/)
+#### Supported operating systems
 
-### Supported Operating Systems
-Create Plugin currently supports the following operating systems:
+Grafana plugin tools work with the following operating systems:
 
 - Linux
 - macOS
 - Windows 10+ with WSL (Windows Subsystem for Linux)
 
-### Choosing a package manager
-Starting from v1.3 of Create Plugin when you create a new plugin, the CLI will use `npm`, `pnpm` or `yarn` depending on which tool you use to run `@grafana/create-plugin`.
+#### Supported Grafana version
+
+We recommend that you build for a version of Grafana later than v9.0. The plugin tools may work with earlier versions, but Grafana Labs may not provide support for them.
+
+#### Recommended tooling
+
+You'll need to have the following tools set up:
+
+- [Go](https://go.dev/doc/install) version 1.18 or later.
+- [Mage](https://magefile.org/).
+- [Node.js](https://nodejs.org/en/download/) version 16 or later.
+  - We recommend that you should install Node.js with all checkboxes related to dependencies checked.
```

Comment:
Is there way to phrase the part about dependencies? It sounds a little awkward. Would this work? 
```suggestion
  - We recommend that you install Node.js with all dependencies.
```

Reply:
What do we mean by _"with all dependencies"_ or _"with all checkboxes related to dependencies checked."_?

Reply:
Might be best to remove this line entirely. Possibly it's left over from older versions of Node that required additional dependencies for it to work.

Reply:
Yeah, I agree, probably that would be the best. 👍